reconcile/gitlab_labeler.py

```diff
@@ -10,6 +11,48 @@
 QONTRACT_INTEGRATION = 'gitlab-labeler'
 
 
+def guess_onboarding_status(changed_paths: List[str]) -> Optional[str]:
+    """
+    Guess the onboarding status of a given MR from the changed paths of the
+    MR.
+    """
+    apps = queries.get_apps()
+    #   logging.info(apps)
+    app_name = get_app_name(changed_paths)
+    if app_name is not None:
+        app = None
+        for a in apps:
+            if a['name'] == app_name:
+                app = a
+
+    if app is not None:
+        logging.info(app['onboardingStatus'])
+        return app['onboardingStatus']
+
+    return None
+
+
+def get_app_name(changed_paths: List[str]) -> Optional[str]:
+    """
+    Guess the service name of a given MR from the changed paths of the
+    MR. This will allow to add the onboarding status to the MR's as label
+    """
+    for path in changed_paths:
+        if 'data/services/' in path:
+            path_slices = path.split('/')
+            try:
+                if path_slices[2] == "insights":
```

Comment:
i think we shouldn't hard code service names here

Reply:
As the parent app pattern is not only on insights, you may find it in other places as ocm or rhobs.

I acknowledge that this is hard, but you must avoid hardcoding and try to come with a series of heuristics that can give you a hint of the service by inspecting the path and not by trusting in positions.

Reply:
I've discussed a possible approach to fix this with @andreugallofre . The idea would be to query qontract-server to have an idea of which apps are children apps or parent apps or none. Having that it would be possible to know which positions are relevant. He will experiment with it.